inbm/dispatcher-agent/dispatcher/remediationmanager/remediation_manager.py

```diff
@@ -74,6 +74,11 @@ def _remove_images(self, ids: Any) -> None:
         for image_id in ids:
             if image_id in self.container_image_list:
                 self._remove_single_image(image_id)
+            else:
+                self._dispatcher_callbacks.broker_core.telemetry('DBS Security issue raised on imageID: '
+                                                                 + str(image_id)
+                                                                 + '.  Image is not present in container image list.')
+
         self.container_image_list[:] = []
```

Comment:
document purpose of self.container_image_list on line 36 and consider choosing a more descriptive variable.  also describe invariants: what does container_image_list contain at all times? how do methods maintain invariant/invariants? i only see a clear and an append in this file.

basically, someone reviewing an individual method or the whole file should be able to follow the logic without jumping around a lot or mentally translating variable names.

Reply:
Renamed container_image_list variable and added a description for the same variable.  Also renamed another variable image_name. 

 **how do methods maintain invariant/invariants? i only see a clear and an append in this file.**
-> Currently we are performing only clear and append on container_image_list.

Reply:
@gblewis1 Could you please provide comments on the latest changes.